readthedocs/projects/tasks.py

```diff
@@ -359,6 +359,11 @@ def run(self, version_pk):  # pylint: disable=arguments-differ
                 update_on_success=False,
                 environment=self.get_rtd_env_vars(),
             )
+            log.info(
+                'Running sync_repository_task: project=%s, version=%s',
```

Comment:
nit:
```suggestion
                'Running sync_repository_task: project=%s version=%s',
```

(just to keep compatibility with others and make parsing logs easier)

Reply:
@humitos we should probably switch to structlog, which does this for us :)

Reply:
I don't know structlog, but I will read about it. In fact, I wanted to suggest to review all our logs for a while, but I don't think we had the time and energy to do that yet. I'd vote +1 to whatever thing that we use that makes our logs better.